### PR TITLE
Create mobile hamburger for the menu

### PR DESCRIPTION
**Pull request by mwafrika**
#### When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.

https://user-images.githubusercontent.com/25848053/150969547-06918935-c167-481d-9091-832bf3b02c0d.mp4

### CODE REVIEW

**Review by lily-coder — COMMENTED** [review]
Hi @mwafrika , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no JavaScript linters. Great job in adding the `.eslintrc.json` file, however, you also need to add the following section to the `.github/worklfows/linters.yml` file under `jobs:` for the linters to work as in this [link](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml)
```
eslint:
    name: ESLint
    runs-on: ubuntu-18.04
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
        with:
          node-version: "12.x"
      - name: Setup ESLint
        run: |
          npm install --save-dev eslint@7.x eslint-config-airbnb-base@14.x eslint-plugin-import@2.x babel-eslint@10.x
          [ -f .eslintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/javascript/.eslintrc.json
      - name: ESLint Report
        run: npx eslint .
```
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/index.html [hunk 1]
```diff
@@ -1,27 +1,38 @@
...
+
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous"
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="index.css" />
     <title>Document</title>
```
Comment (brytebee) [inline src/index.html]: - [ ] **Required:** This title is not a valid title. Kindly change it such that it describes its content. Example Mwafrika's Portfolio, Mwafrika Mufungizi Josué, Mwafrika Mufungizi's Portfolio, etc.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi Team 👋,

Good job so far coming so far with the progress you've made! 💪💪

## Highlights
You have done amazing in the following:
- good use of JS functions✔️
- good interactivity with page animations✔️
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @Team** , 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 

-  When the user clicks (or taps) the hamburger button, the mobile menu appears.✔️  
- When the user clicks (or taps) the close (X) button, the mobile menu disappear.✔️  
- When the user clicks any of the mobile menu options, a correct part of the page is displayed.✔️  
-  When the user clicks any option of the mobile menu, the mobile menu disappear.✔️  
- You have followed JavaScript best practices correctly  ✔️  
- Your Design looks awesome, keep rocking :rocket: 


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**